### DISCUSSION TITLE
Height transition not applying to image element

### DISCUSSION THREAD

**Discussion by hcphoon01** [tailwindlabs/tailwindcss#5826] [Help]
I'm trying to implement a transition on an image so when I hover over it, the image expands with a smooth transition. I've recreated my issue on Tailwind Play [https://play.tailwindcss.com/QfZSly1nEB](https://play.tailwindcss.com/QfZSly1nEB)

As shown, hovering over the image works as expected and the image size changes, however it is not animated as is expected. I've followed the guidance in the docs regarding transition properties but with no luck

**Comment by NovusOpus** [comment]
Not sure what you're trying to do by using the height custom property, but should not scale work better in your case? 

[Tailwind Play Example here](https://play.tailwindcss.com/suYaH8egoF).

But if you need to make the height property work, try to set and specific height in `hover:` like: `hover:h-40` or `hover:h-screen` instead of `hover:h-auto` and it should work fine now. I think what's happening when you use h-auto is the browser overrides tailwind properties so the transition breaks. But I could be wrong or there's a way to fix this to make it work with h-auto.
